pkgs/tools/misc/tmpreaper/default.nix

```diff
@@ -0,0 +1,19 @@
+{ stdenv, fetchurl, autoconf, automake }:
+
+stdenv.mkDerivation rec {
+  pname = "tmpreaper";
+  version = "1.6.17";
+  src = fetchurl {
+    url = "https://deb.debian.org/debian/pool/main/t/tmpreaper/tmpreaper_${version}.tar.gz";
+    sha256 = "1ca94d156eb68160ec9b6ed8b97d70fbee996de21437f0cf7d0c3b46709fecbc";
+  };
+
+  meta = {
+    platforms = [ "x86_64-darwin" "aarch64-darwin" ];
+    homepage = "https://packages.debian.org/sid/tmpreaper";
+    description = "Clean up files in directories based on their age";
+    license = "GPL-2.0-only";
+  };
+
+  buildInputs = [ autoconf automake ];
```

Comment:
I don't think these are needed. The source tarball already contains a suitable `configure` script.

Build tools usually should also go in nativeBuildInputs (when cross compiling, they have to have the architecture of the build machine, not the host machine). The default configure phase doesn't call these either, so this just does nothing.

The `buildInputs` should be above the meta attrs.